geomesa-convert/geomesa-convert-all/src/test/scala/org/locationtech/geomesa/convert/URLConfigProviderTest.scala

```diff
@@ -32,41 +32,73 @@ class URLConfigProviderTest extends Specification {
           req.asInstanceOf[Request].setHandled(true)
           resp.getWriter.write(
             """
-              |geomesa = {
-              |  sfts = [
-              |    {
-              |      type-name = "example-csv-fromurl"
+              |geomesa {
+              |  sfts {
+              |    "example-csv-url" = {
```

Comment:
it's the removal of the '[' that's key - HOCON doesn't merge arrays, but it does merge objects.